tests/test_runs/test_run_functions.py

```diff
@@ -169,6 +191,234 @@ def test_run_and_upload(self):
                 check_res = self._check_serialized_optimized_run(run.run_id)
                 self.assertTrue(check_res)
 
+            # todo: check if runtime is present
+            pass
+
+
+    def test_initialize_model_from_run(self):
+        clf = sklearn.pipeline.Pipeline(steps=[('Imputer', Imputer(strategy='median')),
+                                               ('VarianceThreshold', VarianceThreshold(threshold=0.05)),
+                                               ('Estimator', GaussianNB())])
+        task = openml.tasks.get_task(11)
+        run = openml.runs.run_task(task, clf, avoid_duplicate_runs=False)
+        run_ = run.publish()
+        run = openml.runs.get_run(run_.run_id)
+
+        modelR = openml.runs.initialize_model_from_run(run.run_id)
+        modelS = openml.setups.initialize_model(run.setup_id)
+
+        flowR = openml.flows.sklearn_to_flow(modelR)
+        flowS = openml.flows.sklearn_to_flow(modelS)
+        flowL = openml.flows.sklearn_to_flow(clf)
+        openml.flows.assert_flows_equal(flowR, flowL)
+        openml.flows.assert_flows_equal(flowS, flowL)
+
+        self.assertEquals(flowS.components['Imputer'].parameters['strategy'], '"median"')
+        self.assertEquals(flowS.components['VarianceThreshold'].parameters['threshold'], '0.05')
+        pass
+
+    def test_get_run_trace(self):
+        # get_run_trace is already tested implicitly in test_run_and_publish
+        # this test is a bit additional.
+        num_iterations = 10
+        num_folds = 1
+        task_id = 119
+        run_id = None
+
+        task = openml.tasks.get_task(task_id)
+        # IMPORTANT! Do not sentinel this flow. is faster if we don't wait on openml server
+        clf = RandomizedSearchCV(RandomForestClassifier(random_state=42),
+                                 {"max_depth": [3, None],
+                                  "max_features": [1, 2, 3, 4],
+                                  "bootstrap": [True, False],
+                                  "criterion": ["gini", "entropy"]},
+                                 num_iterations, random_state=42)
+
+        # [SPEED] make unit test faster by exploiting run information from the past
```

Comment:
Not so sure. This is the one thing this function is supposed to test. Let's keep it for now?